### PR TITLE
tests: add new testing file EventType (SDKCF-6378)

### PR DESCRIPTION
**Pull request by ts-robih-robihamanto**
# Description
Increase code coverage in `EventType.swift`
Add a new file for EventTypea and add tests for the name property.

## Links
SDKCF-6378

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data **before every commit**, including API endpoints and keys
- [ ] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by ts-dzmitry-kurski — APPROVED** [review]

**Review by imaginaris — CHANGES_REQUESTED** [review]
please add EventTypeSpec.swift to this project as well `SampleSPM/SampleSPM.xcodeproj`.
And also please remove link to Jira board in the description (it's not a public information)

**Comment by ts-robih-robihamanto** [comment]
> please add EventTypeSpec.swift to this project as well `SampleSPM/SampleSPM.xcodeproj`. And also please remove link to Jira board in the description (it's not a public information)

Added [here](https://github.com/rakutentech/ios-inappmessaging/pull/254/commits/416a64130c0129cad780dc859d2af0afd0f06d81).

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	- Removed "last user" cache to avoid overwriting anonymous user cache during init [SDKCF-6409]
 - Improvements:
 	- Created new test scheme to combine unit test and UI test results [SDKCF-6356]
+    - Added a new file for EventType unit test to increase code coverage [SDKCF-6378]
```
Comment (malayli) [inline CHANGELOG.md]: @ts-robih-robihamanto Please fix this indentation.

**Review by imaginaris — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ts-robih-robihamanto) [reply CHANGELOG.md]: Fixed [here](https://github.com/rakutentech/ios-inappmessaging/pull/254/commits/591b7474ac3645273c7e7763c86fa6ab90e5cd27).

**Review by malayli — APPROVED** [review]

**Pull request merged**